src/extensionFunctions.ts

```diff
@@ -560,8 +560,8 @@ function collectElements(tagname: string) : any[] {
 	if (DidactWebviewPanel.currentPanel) {
 		const html : string | undefined = DidactWebviewPanel.currentPanel.getCurrentHTML();
 		if (html) {
-			const document = new DOMParser().parseFromString(html, 'text/html');
-			const links = document.getElementsByTagName(tagname);
+			const document = parse(html);
```

Comment:
I wonder if we can make one further tweak and use colectElements in the getHeadingText function, just to minimize the repetition?

Reply:
good idea